AsyncDisplayKit/Private/ASInternalHelpers.mm

```diff
@@ -70,3 +70,12 @@ CGFloat ASRoundPixelValue(CGFloat f)
 {
   return roundf(f * ASScreenScale()) / ASScreenScale();
 }
+
+@implementation NSIndexPath (ASInverseComparison)
+
+- (NSComparisonResult)as_inverseCompare:(NSIndexPath *)otherIndexPath
```

Comment:
hmmm...in the layer extensions, we use asyncdisplaykit_ as the prefix.  I guess using asdk_ as @levi suggests would actually be better, but we should do a mass rename of the other properties in a later diff if so.